charts/datagovuk/templates/publish/podmonitor.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.monitoring.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ .Release.Name }}-publish
+  labels:
+    app: {{ .Release.Name }}-publish
+    helm.sh/chart: govuk-dgu-charts
+    app.kubernetes.io/name: publish
+    app.kubernetes.io/managed-by: Helm
```

Comment:
These should come from the template partial in _helpers.tpl, but that also needs fixing in the rest of the chart so thought I'd leave it for now.